### PR TITLE
Add va_chart imagemap

### PR DESCRIPTION
**Pull request by marlycormar**
Addresses enhancement #30.

### CODE REVIEW

**Comment by ChemiKyle** [comment]
The following 'bug' is a result of the table rendering in the default table exports report, clicking the "Table not displaying correctly?" toggle rectifies the rendering error. I am leaving this comment for a record (and because I spent 30 minutes on it before figuring out it was simply a rendering error...), but the value actually recorded in the export is valid. I don't expect study staff would go in and look here, it was fairly tedious to check this way.

It is functional, but the coordinates listed below the image do not match the coordinates recorded as checked. See images below. The value shown as recorded is 2 discrete steps higher on the y axis.

Coordinates below image:  
Screen Shot 2019-06-21 at 8 57 50 AM

Records table:  
Screen Shot 2019-06-21 at 8 55 05 AM

Stepping along the x axis shows similar behavior:  
Screen Shot 2019-06-21 at 9 03 28 AM

Stepping along the y axis also shows similar behavior:  
Screen Shot 2019-06-21 at 9 06 24 AM

**Comment by ChemiKyle** [comment]
For some relevant commentary, I looked into the attribution from #30 and it does appear the in the paper [here](https://ieeexplore.ieee.org/document/6365167#sec4). It's a bit pithy, since Rheumatoid Man gets a description maybe the paper should be linked above the image.

File: README.md [hunk 1]
```diff
@@ -68,6 +68,11 @@ The Rheumatoid man imagemap tool reflects disease activity and progression by re
 ![RHEUMATOID_MAN](./img/rheumatoid_man.png)
 
 
+**`VA_CHART`**
+
```
Comment (ChemiKyle) [inline README.md]:
```suggestion

Information on the VA chart tool can be found in [section 4 of this publication](https://ieeexplore.ieee.org/document/6365167#sec4)  
```

**Comment by marlycormar** [comment]
Thanks for the input @ChemiKyle!!

**Pull request merged**